ftw/crawler/tests/test_purging.py

```diff
@@ -59,3 +59,22 @@ def test_doesnt_touch_any_docs_not_starting_with_site_urls(self, delete):
         purge_removed_docs_from_index(self.config, sitemap_index, indexed_docs)
 
         self.assertEquals(0, delete.call_count)
+
+    @patch('ftw.crawler.solr.SolrConnector.delete')
```

Comment:
Didn't know about `@patch`! Thank you!
https://docs.python.org/3/library/unittest.mock.html#patch